.changeset/common-melons-live.md

```diff
@@ -0,0 +1,7 @@
+---
+'@kaizen/components': major
+---
+
+Rollout of @kaizen/components V2
+
+- Rename Select (Next) to SingleSelect and move to base entry point
```

Comment:
Figured I'd create this here so we can keen adding to in once its merged into the feature branch